drivers/ma_reports/app/models/ma_reports/csg_engage/report_components/program.rb

```diff
@@ -26,12 +26,12 @@ def initialize(program_mapping)
 
     private
 
-    def project
-      @project ||= program_mapping.project
+    def project_ids
+      @project_ids ||= program.program_mappings.pluck(:project_id)
     end
 
     def households_scope
-      project.enrollments.heads_of_households.preload(project: [:project_cocs])
+      GrdaWarehouse::Hud::Enrollment.joins(:project).where(project: { id: project_ids }).heads_of_households.preload(project: [:project_cocs])
```

Comment:
Where is household_scope getting used?  This change potentially will give you back households with the same `HouseholdID` in different projects in the same scope, so that should be disambiguated with `[HouseholdID, ProjectID, data_source_id]` if you are counting households.  That said, HouseholdID is _supposed_ to be unique across a data source, so that's a theoretical need.